spec/system/jobseekers_can_reset_password_spec.rb

```diff
@@ -41,9 +38,7 @@
       fill_in "jobseeker[email]", with: jobseeker.email
       click_on I18n.t("buttons.continue")
       travel_to(3.hours.from_now) do
-        raw_reset_password_token = devise_token_from_last_mail(:reset_password)
-        reset_password_url = edit_jobseeker_password_url(reset_password_token: raw_reset_password_token)
-        visit reset_password_url
+        visit first_link_from_last_mail
```

Comment:
This is nice and can be used in a couple places I think (an alternative approach I used was to visit the path with the token in the param, rather than actually clicking the link in the email)

Reply:
When you merge, I can add this method to your specs. Mine isn't being merged until I chat with Chris.

Reply:
Or you could pre-base with a copy paste of this method.